src/test/java/systemtests/ClearCommandSystemTest.java

```diff
@@ -39,23 +36,6 @@ public void clear() {
         assertCommandSuccess(command, expectedResultMessage, new ModelManager());
         assertSelectedCardUnchanged();
 
-        /* Case: selects first card in person list and clears address book -> cleared and no card selected */
```

Comment:
Yeah, cos now `ClearCommand` clears everything, both persons and tasks, so I moved it to inherit `AppSystemTest` instead of `ContactsSystemTest`, so I don't have `selectPerson`, `showPersonsWithName`, etc. anymore.

Reply:
Ah right got it.